src/executables/uber-price.js

```diff
@@ -25,19 +25,25 @@ const symbolService = new SymbolService();
 program
   .option('-s, --start <start>', 'specify start address')
   .option('-e, --end <end>', 'specify end address')
-  .option('-u, --unit <unit>', 'specify distance unit')
+  .option('-u, --unit [unit]', 'specify distance unit')
   .parse(process.argv);
 
-try {
-  service.executePriceEstimates(program.start, program.end, program.unit)
-    .then(table => console.log(table))
-    .catch((e) => {
-      if (isDistanceExceededError(e)) {
-        console.log(`Maximum distance of ${symbolService.getMaximumDistanceSymbol()}  miles exceeded between start address: ${program.start} and end address: ${program.end}`);
-      } else {
-        console.error('Could not get price estimates');
-      }
-    });
-} catch (e) {
-  console.error('Could not get price estimates');
+
+if (typeof program.start !== 'string' && typeof program.end !== 'string') {
```

Comment:
will `program.start` or `program.end` ever not be `string`s?

I tested the following error cases when I pulled your branch locally

![image](https://user-images.githubusercontent.com/8136030/27619750-bf3992d4-5b92-11e7-9b52-f7d297243b54.png)